File: app/src/main/java/com/gago/david/myland/Models/PlantTypeObject.java
package com.gago.david.myland.models;

import java.io.Serializable;

public class PlantTypeObject implements Serializable {
    public String name;
    public int icon;
    public String color;

    public PlantTypeObject(String name, int icon, String color) {
        this.name = name;
        this.icon = icon;
        this.color = color;
    }

    @Override
    public String toString() {
        return "PlantTypeObject{" +
                "name='" + name + '\'' +
                ", icon=" + icon +
                ", color='" + color + '\'' +
                '}';
    }
}

File: app/src/main/java/com/gago/david/myland/Models/PlantTypeObject.kt
package com.gago.david.myland.models

import java.io.Serializable

class PlantTypeObject(var name: String, var icon: Int, var color: String) : Serializable {
    override fun toString(): String {
        return "PlantTypeObject{" +
                "name='" + name + '\'' +
                ", icon=" + icon +
                ", color='" + color + '\'' +
                '}'
    }
}